packages/bundle/src/initialize.ts

```diff
@@ -67,10 +67,15 @@ export default async (_config, sentry) => {
 
   __root.config = fromJS(cfg);
   __root.sentry = sentry;
-
+  console.log('CFG', cfg);
   /** Setup analytics */
   __root.analytics = AnalyticsDOM(
-    { platform: cfg.platform, key: cfg.key, events: cfg.analytics || {} },
+    {
+      platform: cfg.platform,
+      key: cfg.key,
+      events: cfg.analytics,
+      context: cfg.context || {},
```

Comment:
I think here it is better to not send a context at all if it does not exist like this 
```
{
      platform: cfg.platform,
      key: cfg.key,
      events: cfg.analytics,
      ...(cfg.context && { context: cfg.context }),
 }

Reply:
Not initialiazed if not configured